amgut/db/patches/0032.sql

```diff
@@ -0,0 +1,561 @@
+UPDATE group_questions SET survey_group = 4, display_index = 20 WHERE survey_question_id = 74;
+UPDATE group_questions SET survey_group = 4, display_index = 21 WHERE survey_question_id = 75;
+INSERT INTO group_questions (survey_group, display_index, survey_question_id) VALUES (4, 22, 157);
+UPDATE group_questions SET survey_group = 4, display_index = 23 WHERE survey_question_id = 76;
+
+UPDATE group_questions SET survey_group = 5, display_index = 0 WHERE survey_question_id = 116;
+
+ALTER TABLE group_questions ADD CONSTRAINT idx_group_questions UNIQUE ( survey_group, display_index );
+----------------------------------------------------------
+-- survey_question_response_type
+----------------------------------------------------------
+INSERT INTO survey_question_response_type (survey_question_id, survey_response_type) VALUES
+(148, 'SINGLE'), (149, 'SINGLE'), (150, 'TEXT'), (153, 'SINGLE'),
+(154, 'MULTIPLE'), (155, 'SINGLE'), (156, 'SINGLE'), (157, 'SINGLE'), (158, 'SINGLE'),
+(159, 'SINGLE'), (160, 'SINGLE');
+
+----------------------------------------------------------
+-- survey_response
+----------------------------------------------------------
+INSERT INTO survey_response (american, british) VALUES
+('2+ times/day', '2+ times/day'),
+('1-2 times/day', '1-2 times/day'),
+('Once a day', 'Once a day'),
+('I tend to be constipated (have difficulty passing stool) - Type 1 and 2', 'I tend to be constipated (have difficulty passing stool) - Type 1 and 2'),
+('I tend to have diarrhea (watery stool) - Type 5, 6 and 7', 'I tend to have diarrhea (watery stool) - Type 5, 6 and 7'),
+('I tend to have normal formed stool - Type 3 and 4', 'I tend to have normal formed stool - Type 3 and 4'),
+('Depression', 'Depression'),
+('Bipolar disorder', 'Bipolar disorder'),
+('PTSD (post-traumatic stress disorder)', 'PTSD (post-traumatic stress disorder)'),
+('Schizophrenia', 'Schizophrenia'),
+('Anorexia nervosa', 'Anorexia nervosa'),
+('Bulimia nervosa', 'Bulimia nervosa'),
+('Substance abuse', 'Substance abuse'),
+('Prediabetes', 'Prediabetes'),
+('Type I diabetes', 'Type I diabetes'),
+('Type II diabetes', 'Type II diabetes'),
+('Gestational diabetes', 'Gestational diabetes'),
+('Libya', 'Libya'),
+('Vietnam', 'Vietnam'),
+('Taiwan', 'Taiwan'),
+('No treatment', 'No treatment'),
+('Surgery only', 'Surgery only'),
+('Chemotherapy', 'Chemotherapy'),
+('Radiation therapy', 'Radiation therapy');
+
+----------------------------------------------------------
+-- survey_question_response
+----------------------------------------------------------
+--Update responses from stool quality
+--Add new responses as valid for the question
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES
+(38, 'I tend to be constipated (have difficulty passing stool) - Type 1 and 2', 21),
+(38, 'I tend to have diarrhea (watery stool) - Type 5, 6 and 7', 22),
+(38, 'I tend to have normal formed stool - Type 3 and 4', 23);
```

Comment:
These actually render out of order on my machine.  Type 3 and 4 should be swapped with Type 5, 6, and 7 right?